cli/dcoscli/main.py

```diff
@@ -101,10 +101,10 @@ def _config_log_level_environ(log_level):
     """
 
     if log_level is None:
```

Comment:
If this function does nothing on None, you should probably just have the client code check for None, and then not call